django_redis/client/default.py

```diff
@@ -489,10 +490,10 @@ def ttl(self, key, version=None, client=None):
         if not client.exists(key):
             return 0
 
-        t = client.ttl(key)
+        t = client.pttl(key)
```

Comment:
I would put a warning and highlight in README that `ttl` now returns a float. I would also round it to 3 decimals

Reply:
How should a warning be displaed? Using the warnings.warn function or something else?

Reply:
warning and docs

Reply:
Also a comment on the tests would be helpful

Reply:
I took some time to think about it and I agree with @terencehonles, it would be better to add `pexpire` and `pttl`

Reply:
@abbasidaniyal what do you think? are you able to work on this quickly? I would love to add it to the next release. For the moment I will write the `CHANGELOG` and wait a day or two for your response, otherwise we will release without this change.

Reply:
Yes sure. Working on it

Reply:
@WisdomPill Let me know if I'm on the right path. If not, do let me know what changes I should make!